### PR TITLE
Add full list of errors

### PR DESCRIPTION
**Pull request by dmanchon**
just dummy conversion from macro error expansion to camel case exception in python

### CODE REVIEW

**Comment by ultrabug** [comment]
Hey @dmanchon I find this useful, could you squash those commits maybe?

**Comment by dmanchon** [comment]
> Hey @dmanchon I find this useful, could you squash those commits maybe?

thanks @ultrabug! done!

**Comment by pfreixes** [comment]
Awesome work @dmanchon that is super helpful!!!

Wondering if it would make sense on promoting the `_raise_if_error` function to as part of the `errors.pyx` [1] module and start using it in every place of the driver, for example here [2] and here [3]?

So any time that we call the C++ interface and a `CASS_ERROR` instance is returned we will be delegating to that function the raising of an error.

[1] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/errors.pyx
[2] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/statement/statement.pyx#L69
[3] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/statement/batch.pyx#L22

**Comment by dmanchon** [comment]
> Awesome work @dmanchon that is super helpful!!!
> 
> Wondering if it would make sense on promoting the `_raise_if_error` function to as part of the `errors.pyx` [1] module and start using it in every place of the driver, for example here [2] and here [3]?
> 
> So any time that we call the C++ interface and a `CASS_ERROR` instance is returned we will be delegating to that function the raising of an error.
> 
> [1] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/errors.pyx
> [2] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/statement/statement.pyx#L69
> [3] https://github.com/pfreixes/acsylla/blob/master/acsylla/_cython/statement/batch.pyx#L22

@pfreixes  done, ready for review!

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,8 @@
         sources=["acsylla/_cython/cyacsylla.cpp"],
         include_dirs=[CPP_CASSANDRA_INCLUDE_DIR],
         extra_objects=[CPP_CASSANDRA_STATIC_LIB_DIR],
-        extra_compile_args=["-std=c++11"],
+        extra_link_args=["-g"],
```
Comment (pfreixes) [inline setup.py:24]: I guess that we should remove this, `-g` should be used only for debugging purposes.

File: acsylla/_cython/result/value.pyx [hunk 2]
```diff
@@ -17,39 +17,36 @@ cdef class Value:
     def int(self):
         """ Returns the int value of a column.
 
-        Raises a `ColumnValueError` if the value can not be retrieved"""
+        Raises an Exception if the value can not be retrieved"""
```
Comment (pfreixes) [inline acsylla/_cython/result/value.pyx:20]: Could we say something like

```
Raises a `CassException` derivated class if the value can not be retrieved
````

The same for other places where. Also, we would need to modify the interfaces documentation if its needed [1]

[1] https://github.com/pfreixes/acsylla/blob/master/acsylla/base.py#L176

File: setup.py [hunk 3]
```diff
@@ -21,7 +21,8 @@
...
+        extra_compile_args=["-std=c++11", "-g"],
```
Comment (pfreixes) [inline setup.py:25]: the same here with the `-g`

**Review by pfreixes — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (dmanchon) [reply setup.py:24]: ups, sorry about that! i was trying to debug it (unsuccesfully :P)

File: acsylla/_cython/result/value.pyx (hunk 2 above)
Comment (dmanchon) [reply acsylla/_cython/result/value.pyx]: sure. done.

File: setup.py (hunk 1 above)
Comment (dmanchon) [reply setup.py]: done

File: setup.py (hunk 3 above)
Comment (dmanchon) [reply setup.py]: done

**Review by pfreixes — APPROVED** [review]
Thanks for the work done here, congrats for the first PR merged to the project!

**Comment by ultrabug** [comment]
Thanks @dmanchon !

**Pull request merged**